src/components/Login/Login.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { BaseColor } from "../../config/color";
+import { Type } from "../../shared/contexts/actions";
+import { useGeneralContext } from "../../shared/contexts/StoreProvider";
+import { useFindUser } from "../../shared/graphql/request/userRequest";
+import Button from "../Button/Button";
+
+const LoginComponent = () => {
+  const [email, setEmail] = useState("");
+  const [error, setError] = useState("");
+  const { findUser, errorRequest, data, loading } = useFindUser();
+
+  const { dispatch } = useGeneralContext();
+
+  useEffect(() => {
+    if (errorRequest?.message) {
+      return setError(errorRequest?.clientErrors + "");
+    }
+
+    if (data !== undefined && data?.users.length !== 0) {
+      dispatch({ type: Type.LOGIN, payload: data?.users[0] });
+    }
+    if (data?.users.length === 0) {
+      setError("Email not register");
+    }
+  }, [data, errorRequest, dispatch]);
+
+  const handleClickLogin = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    try {
+      if (!isValidEmail(email)) {
+        return setError("Email not valid");
+      }
+      setError("");
+      await findUser(email);
+    } catch (e) {
+      console.log("ERROR CAtCH", e);
```

Comment:
how does the use knows something wrong happened with this request?

Reply:
because in my useEffect it also compares the errorRequest and if this happens. I put the value of error into my useState and this state informs the user